spec/api-tray-spec.js

```diff
@@ -24,6 +24,23 @@ describe('tray module', () => {
     })
   })
 
+  describe('tray.popUpContextMenu', () => {
+    before(function () {
```

Comment:
i think this might only be needed on windows? it's failing on macOS as well.

Reply:
It seems that this API is blocking on macOS. I have changed the test to only run on Windows.